protocol/migrations/sqlite/1697038980_add_timestamp_hash_ratchet.up.sql

```diff
@@ -0,0 +1,28 @@
+
+CREATE TABLE hash_ratchet_encryption_v2 (
+  group_id BLOB NOT NULL,
+  key_id INT NOT NULL,
+  key BLOB NOT NULL,
+  key_timestamp BLOB,
+  hash_id BLOB NOT NULL,
```

Comment:
```suggestion
  key_id BLOB NOT NULL,
```
To be consistent with code naming.